dask/dataframe/io/tests/test_parquet.py

```diff
@@ -465,3 +461,14 @@ def test_to_parquet_lazy(tmpdir, get):
     ddf2 = dd.read_parquet(tmpdir)
 
     assert_eq(ddf, ddf2)
+
+
+def test_empty(fn):
+    df = pd.DataFrame({'a': ['a', 'b', 'b'], 'b': [4, 5, 6]})[0:0]
+    ddf = dd.from_pandas(df, npartitions=2)
+    ddf.to_parquet(fn, write_index=False)
+    files = os.listdir(fn)
+    assert '_metadata' in files
+    out = dd.read_parquet(fn).compute()
+    assert len(out) == 0
+    assert list(out.columns) == ['a', 'b']
```

Comment:
For some reason, I was assuming it wouldn't work for empty dataframes, but it's fine.